core/domain/classifier_services.py

```diff
@@ -384,13 +351,13 @@ def fetch_next_job():
     return next_job
 
 
-def store_classifier_data(job_id, classifier_data):
+def store_classifier_data(job_id, classifier_data_proto):
     """Checks for the existence of the model and then updates it.
 
     Args:
         job_id: str. ID of the ClassifierTrainingJob domain object.
-        classifier_data: dict. The classification model which needs to be stored
-            in the job.
+        classifier_data_proto: object. The job result of training job
```

Comment:
See notes above re 'object'

Reply:
Here, it is a protobuf object but the type of it won't be known in advance as it depends on `algorithm_id`. For example in case of `TextClassifier`, this will be `TextClassifierFrozenModel` and this would be different for different classifiers. What should be written in that case?

Reply:
Can you have those frozen models be a subclass of BaseFrozenProtobufModel or similar, and then use BaseFrozenProtobufModel as the type? Then in the docstring, explain it's a subclass of that. This provides way more context than just "object".

Reply:
Unfortunately, protocol buffers don't support inheritance of message type, hence defining a common base class won't be possible. An an alternative perhaps we can use `FrozenModel` to refer to different types of frozen models?

Reply:
Yeah, I think that will have to do. In that case, have a readme in core/domain/proto that explains FrozenModel so that if anyone greps for the type, they'll see it and can understand what it refers to.

Reply:
Done

Reply:
Not done?

Reply:
Done now.